soc/renesas/rz/rzv2h/cr8/soc.c

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2025 Renesas Electronics Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief System/hardware module for Renesas RZ/V2H Group
+ */
+
+#include <bsp_api.h>
+#include <zephyr/init.h>
+#include <zephyr/sys/barrier.h>
+#include <zephyr/drivers/interrupt_controller/gic.h>
+#include <zephyr/irq.h>
+#include <zephyr/cache.h>
+
+/* System core clock is set to 800 MHz after reset */
+uint32_t SystemCoreClock BSP_SECTION_EARLY_INIT;
+
```

Comment:
Porting some fsp irq-related variables is missing

```suggestion
/* System core clock is set to 800 MHz after reset */
uint32_t SystemCoreClock BSP_SECTION_EARLY_INIT;
void * gp_renesas_isr_context[BSP_ICU_VECTOR_MAX_ENTRIES + BSP_CORTEX_VECTOR_TABLE_ENTRIES];
uint16_t g_current_interrupt_num[32];
uint8_t  g_current_interrupt_pointer;
```